words.html

```diff
@@ -64,234 +64,237 @@
+Z każdym przeciwnikiem zagrasz od 3 do 7 rund.
+(Nie będziesz z góry wiedzieć która runda jest ostatnia)
+Czy możesz im zaufać? A raczej... czy oni mogą zaufać <i>Tobie?</i>
 </p>
 
 <p id="iterated_intro_btm">
-Pick your first, <i>real</i> move. <b>Choose wisely.</b>
+Wybierz swój pierwszy, <i>prawdziwy</i> ruch. <b>Wybierz mądrze.</b>
 </p>
 
 <p id="iterated_info_1">
-opponent: [X] of [Y]
+przeciwnik: [X] z [Y]
 </p>
 </p>
 <p id="iterated_info_2">
-your total score: 
+twój łączny wynik:
 </p>
 
 <p id="iterated_score_start">
-And your total score is...
+Twój łączny wynik to...
 </p>
 <p id="iterated_score_1">
-which is, wow, actually <i>impressively</i> bad.
+który jest, łał, szczerze mówiąc <i>imponująco</i> słaby.
 </p>
 <p id="iterated_score_2">
-which, uh, could be worse!
+który, um, mógłby być gorszy!
 </p>
 <p id="iterated_score_3">
-which ain't bad!
+który nie jest taki zły!
 </p>
 <p id="iterated_score_4">
-which is pretty good!
+który jest całkiem niezły!
 </p>
 <p id="iterated_score_5">
-which is <i>perfect!</i> Congrats you have too much time on your hands.
+który jest <i>idealny!</i> Gratulacje, chyba masz za dużo wolnego czasu.
 </p>
 <p id="iterated_score_x">
-...i have no idea how you did that.
+...nie mam pojęcia jak to zrobiłeś/aś.
 </p>
 <p id="iterated_score_end">
-(the lowest &amp; highest possible scores are 7 and 49, respectively)
+(najwyższy wynik to 7, najniższy to 49)
 </p>
 
 <p id="who_were">
-So who were these strange characters you just played against?
+Okej, co to za dziwne postacie przeciw którym właśnie grałeś/grałaś?
 </p>
 
 <p id="character_tft">
-<b>COPYCAT:</b>
-Hello! I start with Cooperate, and afterwards, I just copy whatever you did in the last round. Meow
+<b>PAPUGA:</b>
+Hejka! Zaczynam od współpracy, a potem po prostu kopiuję twój ruch z poprzedniej rundy.
+A tak w ogóle to Polly chce ciasteczko.
 </p>
 
 <p id="character_all_d">
-<b>ALWAYS CHEAT:</b>
-<i>the strong shall eat the weak</i>
+<b>NOTORYCZNY OSZUST:</b>
+<i>silniejsi pożrą słabszych</i>
 </p>
 
 <p id="character_all_c">
-<b>ALWAYS COOPERATE:</b>
-Let's be best friends! &lt;3
+<b>UFNY WSPÓLNIK:</b>
+Bądźmy przyjaciółmi! &lt;3
 </p>
 
 <p id="character_grudge">
-<b>GRUDGER:</b>
-Listen, pardner. I'll start cooperatin', and keep cooperatin',
-but if y'all ever cheat me, I'LL CHEAT YOU BACK 'TIL THE END OF TARNATION.
+<b>MŚCICIEL:</b>
+Słuchaj, kolego, będę współpracować,
+ale jeśli kiedykolwiek mnie oszukasz, BĘDĘ OSZUKIWAŁ DO KOŃCA ŚWIATA I JEDEN DZIEŃ DŁUŻEJ.
 </p>
 
 <p id="character_prober">
-<b>DETECTIVE:</b>
-First: I analyze you. I start: Cooperate, Cheat, Cooperate, Cooperate.
-If you cheat back, I'll act like <span class="tft">Copycat</span>.
-If you never cheat back, I'll act like <span class="all_d">Always Cheat</span>, to exploit you.
-Elementary, my dear Watson.
+<b>DETEKTYW:</b>
+Najpierw będę Cię analizował. Zacznę od: współpraca, oszustwo, współpraca, współpraca.
```

Comment:
Na pewno mi nie brzmi
`+Najpierw będę Cię analizował.` To mi się kojarzy z jakąś obdukcją trupa :D 

Może:
`Najpierw przeprowadzę analizę. Zacznę od: współpraca, oszustwo, współpraca, współpraca.`
`Najpierw przeanalizuję jak reagujesz. Zacznę od: współpraca, oszustwo, współpraca, współpraca.`
`Najpierw przeanalizuję Twoje reakcje. Zacznę od: współpraca, oszustwo, współpraca, współpraca.`